src/lib/block/kuznyechik/kuznyechik.cpp

```diff
@@ -0,0 +1,4520 @@
+                                  {0x8c2fe00adcb3b948, 0x4ad445a6da2da34b}, {0x7d45214f405e1168, 0x14739dfec439382a},
+                                  {0xe15b8f8b33800bf5, 0x632e4c1e4acdba28}, {0x1f4c3fc7e8b67181, 0xd57b90d1141853ff},
+                                  {0x08f92767b6d7b08b, 0xffa54a232a8df1c5}, {0x334a762d98311cc7, 0xcebc148e8343792b},
+                                  {0xf7584afe0b22dcd6, 0x8fac0ed0e001af42}, {0xf16ac1459ceda820, 0x5ea7d8581e149b61},
+                                  {0x17b518a05e61c10a, 0x2adedaf23e95a23a}, {0xadfba18027aa2a08, 0xf6593616e6055689},
+                                  {0x0000000000000000, 0x0000000000000000}, {0x3bb3514a2ee6ac4c, 0x31195eada9ce88ee},
+                                  {0xaa7ff7eed6a648d7, 0xe10e78a28c49d5fa}, {0xfe17b04cdb367a74, 0xb655dccf5ed5e9d7}},
+                                 {{0xc397111b1518eccc, 0x9b933a773e7d9af0}, {0xcd0591a4324c5203, 0x23a6ad710e447bbd},
+                                  {0xd8399218a99a0c17, 0x74027d7846f20aad}, {0x3c2c688604ab43a5, 0x98407bdb17358384},
+                                  {0x64bf1372463d39d4, 0x40ea1b794e18c451}, {0x4ec715c9b35285fc, 0xee61786bdeb72671},
+                                  {0x53b94c0bdbf4e4d6, 0xc32e6c27fb609687}, {0xfb9a54a2898b5176, 0x3e47236ffe999807},
+                                  {0x3db5f2f922a59dd2, 0x0665829b529032e8}, {0xf660b99cc4cd4e20, 0xe71d7ca9018c69fe},
+                                  {0x4daf7848d9402465, 0x8f0eb0ab119b36c5}, {0x18c6ee82d6904142, 0x8efe8fcfb7a380e9},
+                                  {0x76ca50702fc13846, 0x4bb561731e532486}, {0xc20e8b64331632bb, 0x05b6c3377bd82b9c},
+                                  {0x4474b8490c3ec52c, 0x6bc08dae395f464f}, {0x626fc9b39219b825, 0x8234483a1340e4fa},
+                                  {0x7372e73091f7182e, 0xe804faf08c271499}, {0xdd81255817ac2c7f, 0xd7b3e6fbd4863ab2},
+                                  {0xe034d7a13509b1ad, 0xd1d664608616085a}, {0x590ae18b6498a406, 0x468f99e21c88f6b9},
+                                  {0xc8bd26e48c7a726b, 0x801736f29c304ba2}, {0x9c4d2a51a5a4c93b, 0x1fc2f0d67fad4ce2},
+                                  {0x32bee83923fffd6a, 0x2075ecdd270c62c9}, {0x2782eb85b829a37e, 0x77d13cd46fba13d9},
+                                  {0x3f4405076eb9e23c, 0xf92fb31bd8199330}, {0x304f1fc76fe38284, 0xdf3fdd5dad85c311},
+                                  {0x7aa927314489f967, 0x0ccac7f5a4e36413}, {0x8a19446c546036b6, 0x2909e81ff8372d46},
+                                  {0x223a5cc5061f8316, 0xd460a757fdce23c6}, {0x825b1e12a7100988, 0x53e22c5a9556eca0},
+                                  {0x88e8b392187c4958, 0xd643d99f72be8c9e}, {0x6107a432f80b19bc, 0xe35b80fadc6cf44e},
+                                  {0x8fa1f32cea5616de, 0x8ab8739c6a431d59}, {0x46854fb74022bac2, 0x948abc2eb3d6e797},
+                                  {0xaad2ef571e63ca4e, 0x02237ec88f
```

Comment:
Should be `uint8_t`

Reply:
Changed. Will be in the next update to this PR.